### DISCUSSION TITLE
Editing entrypoint.sh after making container out of the image

### DISCUSSION THREAD

**Discussion by johnny-brav0** [lllyasviel/Fooocus#3185] [Q&A] [answered]
Hi,
I dowloaded the docker image provided `ghcr.io/lllyasviel/fooocus` and wanted to do some changes to the code. But it seems the `/content/app` is not mounted on any directory in the local system. Also the `entrypoint.sh` is copied to `/content` which I am unable to modify.

I also built the image on my local system and even tried editing the Dockerfile but nothing seems to be working out so far.

I'm using podman instead of docker but I don't think this should make any difference

What I want to do it be able to edit the code and restart the container with the new code. I appear to be stuck in achieving. It'd be really helpful if someone can point me in the right direction here.

Thanks for help in advance :)

**Comment by mashb1t** [comment, marked as answer]
you'll have to either mount the file yourself to `/content/entrypoint.sh` or build the container again after modifying it. Make sure you use LF instead of CRLF for line breaks when saving in any editor to ensure compatibility with Linux through Docker.

**Reply by johnny-brav0** [reply]
Okay, thanks for the info my friend. I just wanted to know that I'm not reinventing the wheel but it seems this wheel hasn't been invented yet.
Cheers.
